packages/cardano-services/environments/.env.sanchonet

```diff
@@ -1,4 +1,4 @@
 HANDLE_POLICY_IDS=${HANDLE_POLICY_IDS:-f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a}
 OGMIOS_PORT=${OGMIOS_PORT:-1341}
 METADATA_FETCH_MODE="direct"
-SMASH_URL="http://cardano-smash:3100/api/v1"
```

Comment:
Is SMASH_URL not used anywhere?

Reply:
It was removed in #1056

Reply:
Actually the only available smash server is for mainnet, that's why we can't use for other networks